Celeste.Mod.mm/Mod/Meta/MapMeta.cs

```diff
@@ -443,6 +503,16 @@ public CheckpointData Convert()
                 }
             }
         }
+
+        public void AddTo(MapMetaCheckpointData other) {
```

Comment:
Oh, right. I implemented AddTo for checkpoint data before I realized it was useless. Maybe in the future it will not be useless!